cmd/snap-bootstrap/cmd_initramfs_mounts.go

```diff
@@ -1734,7 +1711,12 @@ func generateMountsModeRun(mst *initramfsMountsState) error {
 			}
 		}
 	}
+	// // 4.2->
+	// mount essential snaps for run mode
+	return generateSnapMountsModeRun(mst, isClassic, hasSeedPart, rootfsDir)
+}
 
+func generateSnapMountsModeRun(mst *initramfsMountsState, isClassic, hasSeedPart bool, rootfsDir string) error {
```

Comment:
nitpick: this has a bit too many bool args

Reply:
Ideally I'd extend `initramfsMountsState` struct to do more, so we do not pass this many things.
But fair point